ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorUtil.java

```diff
@@ -337,6 +337,7 @@ private static CompactionResponse requestCompaction(CompactionInfo ci, String ru
     }
     compactionRequest.setInitiatorVersion(ci.initiatorVersion);
     compactionRequest.setPoolName(ci.poolName);
+    compactionRequest.setOrderByClause(ci.orderByClause);
```

Comment:
Will enabling this compactionRequest.setOrderByClause affect hive acid table compaction as well?

Reply:
Yes, it affect Hive ACID compaction as well because compaction implementation is common for both ACID and Iceberg tables. But I believe this change is not dangerous because it just copies an order by clause from compaction info object to the compaction request which was missing; if the order by clause is not defined, it will set it as null. Hive ACID compaction also supports sorting. https://github.com/apache/hive/pull/4077

Reply:
Thats great. Thanks!

I was worried about the side effect this would have on hive acid compactions. This clears it up.

Reply:
Rest of the code looks good to me!

Reply:
Thanks!